topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,258 @@
+>    - "Remove masked bins from the matrix" to `True`
+>    - "Chromosomes to include (and order to plot in)" to `chr2L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr2R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chrX`
+>
+>    > ### {% icon tip %} Tip: log1p
+>    >
+>    > Because of the large differences in counts found int he matrix, it is better to plot the counts using the *–log1p* option.
+>    {: .tip}
+>
+>    - **Plot the log1p of the matrix values**: True
+>
+{: .hands_on}
+
+# Correction of Hi-C matrix
+
+[hicCorrectMatrix](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicCorrectMatrix.html#hiccorrectmatrix) corrects the matrix counts in an iterative manner. For correcting the matrix, it’s important to remove the unassembled scaffolds (e.g. NT_) and keep only chromosomes, as scaffolds create problems with matrix correction. Therefore we use the chromosome names (1-19, X, Y) here. **Important**: Use ‘chr1 chr2 chr3 etc.’ if your genome index uses chromosome names with the ‘chr’ prefix.
+
+Matrix correction works in two steps: first a histogram containing the sum of contact per bin (row sum) is produced. This plot needs to be inspected to decide the best threshold for removing bins with lower number of reads. The second steps removes the low scoring bins and does the correction.
+
+> ### {% icon hands_on %} Hands-on: Matrix diagnostic
+> 
+> 1. **hicCorrectMatrix** {% icon tool %}: Run hicCorrectMatrix adjusting the parameters:
+>    - "Range restriction (in bp)" to `Diagnostic plot`
+>    - "Chromosomes to include (and order to plot in)" to `chr2L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr2R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chrX`
+>
+{: .hands_on}
+
+The output of the program prints a threshold suggestion that is usually accurate but is better to revise the histogram plot. The threshold is visualized in the plot as a black vertical line.
+
+In our case the distribution describes the counts per bin of a genomic distance. To remove all bins with a z-score threshold less / more than X means to remove all bins which have less / more counts than X of mean of their specific distribution in units of the standard deviation. Looking at the distribution, we can select the value of -1.6 (lower end) and 1.8 (upper end) to remove. This is given by the –filterThreshold option in hicCorrectMatrix set to 'correct matrix' mode.
+
+> ### {% icon hands_on %} Hands-on: Matrix correction
+>
+> 1. **hicCorrectMatrix** {% icon tool %}: Run hicCorrectMatrix on the output from previous step adjusting the parameters:

```

Comment:
on which one? the merged bin matrix or the original matrix?

... upon further review: it only works when you use the original matrix. Mention this